memgpt/config.py

```diff
@@ -84,6 +87,14 @@ class MemGPTConfig:
     # version (for backcompat)
     memgpt_version: str = None
 
+    # logging (for logger)
+    logging_level: str = "CRITICAL"  # default log level
```

Comment:
Could we move logging information into a `logging_constants.py` or the `constants.py` file? Since it'll be developers modifying these fields, and we'd like to keep the config file minimal.

Reply:
@sarahwooders I can change it to code, but I will have to abandon this pull request and delete this branch. I will redo it in a new branch and pull the request. Or change to "logging.config" as the config file. 

I hate having multiple configuration files for reasons like configuring Spring Boot for Java web applications. There are so many config files. Setting it up properly requires finding the correct file for the configuration option you need to change.

It is still better to load from a configuration rather than change source files to change. This is better for maintaining code stability and keeping setting out of the code. This will prevent you from accidentally committing debug code to production. This comes from my DevOps Engineering experience and accidentally dealing with bad releases from developers.